### ISSUE TITLE
Autimatically include higher-ordered / higher-ranking communities or townships when searching for an address

### ISSUE THREAD

**Issue by wehkah** [osmandapp/OsmAnd#19835]
### Describe the idea (required)

Now and then, searching an address in Osmand provides no result, because OpenStreetMap assigned the address to a wrong part of a city, the wrong district, etc. (No offence meant, mistakes happen!)

Would it be possible to automatically switch to the next higher ordered community if a search does not find the desired address?

### Tell us about the expected behaviour (required)

The expected behaviour would look like this: I enter an address, which is assigned to a faulty district of the higher ordered city, but Osmand will find the street / the address nonetheless, e.g. bases on the zip code ("Postleitzahl" in Germany), or based on distance.

### Tell us about alternatives you've considered (required)

There are two alternatives: I can use Google Maps to find the desired address, or I can use the phone and ask for manual routing instructions, which feels sooooo like in the 1980s again... :(

**Comment by vshcherb** [comment]
Yes, probably we can combine other data sources to have better result
